src/libraries/System.Text.RegularExpressions/tests/Regex.EscapeUnescape.Tests.cs

```diff
@@ -35,6 +42,12 @@ public void Escape_NullString_ThrowsArgumentNullException()
         public void Unescape(string str, string expected)
         {
             Assert.Equal(expected, Regex.Unescape(str));
+
+            if (expected.Length > 0)
+            {
+                const int Count = 100;
```

Comment:
I guess the 100 takes it out of some stackalloc path?

Reply:
Yes